### DISCUSSION TITLE
🚀 FR make it easier to share 'previews' of entries.

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#4104] [Feature Requests]
### Suggest an idea for ExpressionEngine

I think we discussed this at some point, but it would be nice to have a way to share a preview link, preferably on the frontend.  Or even in the cp, with the old 'eye' icon that would take you directly into the entry with the preview already opened- and be able to link to that.  

There are two ways to preview now- the slickest being the live preview in the cp, using the livepreview conditional in tags to allow say closed entries to show in preview only.  The other ways is the old school way you did it on the frontend where you use role ids to conditionally display 'draft' versions on the frontend.  It would be nice to have both without using a ton of conditionals or different templates to get there.

**Comment by intoeetive** [comment]
@robinsowell how would it be different from actual entry page using preview template?

The Live Preview functionality requires POST request to build the page, so we can't really make that shareable

**Reply by robinsowell** [reply]
They basically want a sharable link not via the cp to preview on the frontend- kind of like how the old pre-live preview 'preview' worked.  Which is easy enough to setup with conditionals- you just use say a role_id in the conditional vs the live_preview conditional.  They're actually looking at the preview on the frontend, so no need for post or anything.  It's not 'live' but it's a preview of content that's not generally accessible.  They're doing it now with Structure, which does link t the 'page' whether it's open, future dated, whatever.  It's not all that hard to just figure out what the url would be, but it's still nice to have a link to the 'page' the entry should show up on if all of the parameters allow it.  And then you futz with the parameters to allow it for people with permission.
